diff --git a/scripts/version.sh b/scripts/version.sh
index 290e97e389..163722dee9 100755
--- a/scripts/version.sh
+++ b/scripts/version.sh
@@ -68,7 +68,7 @@ VERSION_ROOT="v0.15.0"
 VERSION_HELM_JOB="v0.9.15-build20260324"
 
 GO_VERSION_URL="https://raw.githubusercontent.com/kubernetes/kubernetes/${VERSION_K8S}/.go-version"
-VERSION_GOLANG="go"$(curl -sL "${GO_VERSION_URL}" | tr -d '[:space:]')
+VERSION_GOLANG=${VERSION_GOLANG:-"go$(curl -sL "${GO_VERSION_URL}" | tr -d '[:space:]')"}
 
 if [[ -n "$GIT_TAG" ]]; then
     if [[ ! "$GIT_TAG" =~ ^"$VERSION_K8S"[+-] ]]; then
