diff --git a/scripts/build b/scripts/build
index 3c9bc511d6..a0c2a494f7 100755
--- a/scripts/build
+++ b/scripts/build
@@ -162,13 +162,7 @@ INSTALLBIN=$(pwd)/bin
 if [ ! -x ${INSTALLBIN}/cni${BINARY_POSTFIX} ]; then
 (
     echo Building cni
-    TMPDIR=$(mktemp -d)
-    trap cleanup EXIT
-    WORKDIR=$TMPDIR/src/github.com/containernetworking/plugins
-    git clone --single-branch --depth=1 --branch=$VERSION_CNIPLUGINS https://github.com/rancher/plugins.git $WORKDIR
-    cd $WORKDIR
-    rm -rf plugins/meta/flannel
-    git clone --single-branch --depth=1 --branch=$VERSION_FLANNEL_PLUGIN https://github.com/flannel-io/cni-plugin.git plugins/meta/flannel
+    cd "$CNIPLUGINS_DIR"
     sed -i 's/package main/package flannel/; s/func main/func Main/' plugins/meta/flannel/*.go
     GO111MODULE=off GOPATH=$TMPDIR CGO_ENABLED=0 "${GO}" build -tags "$TAGS" -gcflags="all=${GCFLAGS}" -ldflags "$VERSIONFLAGS $LDFLAGS $STATIC" -o $INSTALLBIN/cni${BINARY_POSTFIX}
 )
